pkg/migration/generate.go

```diff
@@ -1152,6 +1158,17 @@ func initRuntimeOptions(cluster *corev1.StorageCluster) {
 	}
 }
 
+func getStorageClusterNameFromClusterID(clusterID string) string {
+	// Must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character
```

Comment:
Does the regex work for `.`? I don't see how.

Reply:
yes it works with `\\.`. Get the regex from the log, and verified the regex in a tester.